hack/make-bundle-vars.mk

```diff
@@ -102,6 +102,13 @@ ROOK_BUNDLE_IMG_TAG ?= v4.15.0
 ROOK_BUNDLE_IMG_LOCATION ?= quay.io/ocs-dev
 ROOK_BUNDLE_IMG ?= $(ROOK_BUNDLE_IMG_LOCATION)/$(ROOK_BUNDLE_IMG_NAME):$(ROOK_BUNDLE_IMG_TAG)
 
+# To be changed when odf-prometheus-operator bundle exists
+PROMETHEUS_BUNDLE_NAME ?= odf-prometheus-operator
+PROMETHEUS_BUNDLE_IMG_NAME ?= $(PROMETHEUS_BUNDLE_NAME)-bundle
+PROMETHEUS_BUNDLE_IMG_TAG ?= prom_1
```

Comment:
@ezio-auditore The image tag should be same as a version. it should be `v4.10.0`

Reply:
Updated